setup.py

```diff
@@ -32,37 +32,43 @@ def merge_req_lists(req_lists: Iterable[List[str]]) -> List[str]:
     "typing_extensions>=4.0.0",
 ]
 
-sqlite_dependencies = [
-    "sqlalchemy>=1.4.27",
+async_files_dependencies = [
+    "aiofiles>=22.1.0",
 ]
 
 redis_dependencies = [
-    "redis>=4.1.2",
+    "aioredis>=2.0.1",
 ]
 
 mongodb_dependencies = [
-    "pymongo==4.3.2",  # TODO: wait for bson using bug will be fixed
-    "bson>=0.5.10",
+    "motor>=3.1.1",
+]
+
+sqlite_dependencies = [
+    "aiosqlite>=0.18.0",
+    "sqlalchemy[asyncio]>=1.4.27",
```

Comment:
This suggestion is not relevant to this PR, maybe it should be done in another one:

We should separate `sqlalchemy` into its own dependency and add it to sqlite, mysql, postgresql using `merge_req_lists`.